.github/workflows/minimum_dependency_checker.yml

```diff
@@ -0,0 +1,93 @@
+name: Minimum Dependency Checker
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'requirements.txt'
+      - 'test-requirements.txt'
+      - 'dask-requirements.txt'
+      - 'koalas-requirements.txt'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+      - name: Run min dep generator - test reqs
+        id: min_dep_gen_test
+        uses: alteryx/minimum-dependency-generator@v2.1
```

Comment:
Do we want to point to v2.1 specifically, or use a generic v2 tag that points to the latest 2.x.y version (we can update the v2 tag to point to new releases as we add them)

Reply:
Fixed to point to v2 tag.